File: rust/src/model/metrics/statistics.rs

#[derive(Debug, Clone, Copy)]
pub struct Statistics {
    pub total_count: usize,
    pub total_duration: f64,
    pub ops_per_sec: f64,
    pub min: f64,
    pub p50: f64,
    pub p75: f64,
    pub p95: f64,
    pub p99: f64,
    pub p99_9: f64,
    pub max: f64,
    pub mean: f64,
    pub std_dev: f64,
}

File: golang/model/metrics/statistics.go
package metrics

type Statistics struct {
	TotalCount    int
	TotalDuration float64
	OpsPerSec     float64
	Min           float64
	P50           float64
	P75           float64
	P95           float64
	P99           float64
	P999          float64
	Max           float64
	Mean          float64
	StdDev        float64
}
